app/controllers/compute_resources_controller.rb

```diff
@@ -63,6 +68,11 @@ def update
     else
       process_error
     end
+  rescue StandardError => e
+    logger.error "Error while updating ovirt resource: #{e.message}\n#{e.backtrace.join("\n")}"
```

Comment:
This isn't necessarily an ovirt resource

Reply:
see my comment above

Reply:
We are rescuing from ` StandardError => e` which is not ovirt specific, therefore the message should not have `ovirt`, Am I reading this right @tbrisker?

Reply:
exactly, since the controller is generic for all compute resources, if for example this is a libvirt resource it doesn't make sense to log an error referring to ovirt.

Reply:
but shouldn't this also only catch ovirt errors instead of any generic ones like the other method?